snapcraft/internal/elf.py

```diff
@@ -589,3 +601,17 @@ def get_elf_files(root: str,
                 elf_files.add(ElfFile(path=path))
 
     return frozenset(elf_files)
+
+
+def get_elf_files_to_patch(elf_files):
```

Comment:
This function plays an important role here, and would benefit from a good docstring.